lib/symbolic/symbolic.rb

```diff
@@ -41,6 +52,10 @@ module ArrayCommand
             # commands depends on the type of the command.
             attr_reader :input
 
+            attr_reader :keep
```

Comment:
Add comment, something like "Indicates if result should be kept on the GPU for further processing."